container/container_unix.go

```diff
@@ -168,6 +169,21 @@ func (container *Container) SecretMountPath() string {
 	return filepath.Join(container.Root, "secrets")
 }
 
+// ConfigsDirPath returns the path to the directory where configs are stored on
+// disk.
+func (container *Container) ConfigsDirPath() string {
```

Comment:
This is where they are stored on the host filesystem, for mounting into the container.